### PR TITLE
[MDCT-2222] - Spruce up footer to be inline with other apps

### PR DESCRIPTION
**Pull request by ntsummers1**
## Description
The MCR Footer isn't quite inline with the other applications. This changes it so that its more or less the same as how Carts does it.

### How to test
./dev local
Look at the bottom of the page and check it out!

## Code author checklist
- [x] I have performed a self-review of my code
- [x] I have added [thorough](https://bit.ly/3zPrxuZ) tests
- [x] I have added analytics, if necessary
- [x] I have updated the documentation, if necessary

## Reviewer checklist (two different people)
- [ ] I have done the deep review and verified the items in the above checklist are g2g
- [ ] I have done the lgtm review

### CODE REVIEW

File: services/ui-src/src/components/layout/Footer.tsx [hunk 1]
```diff
@@ -9,48 +9,57 @@ import mcrLogo from "assets/logos/logo_mcr_footer.png";
...
-                    <Image src={hhsLogo} alt="HHS logo" sx={sx.hhsLogo} />
+            </Box>
+            <Box sx={sx.footerTopRightContainer}>
+              <Flex sx={sx.footerTopRightTopFlex}>
+                <Box sx={sx.footerCMSBrandingLeft}>
+                  <Box sx={sx.hhsLogo}>
+                    <img
+                      src={hhsLogo}
+                      alt="Department of Health and Human Services, USA"
+                    />
```
Comment (braxex) [inline services/ui-src/src/components/layout/Footer.tsx:32]: Suggestion: Recommend using Chakra's `Image` component here and elsewhere in PR for consistency.

File: services/ui-src/src/components/layout/Footer.tsx [hunk 2]
```diff
@@ -105,57 +114,93 @@ const sx = {
...
-  footerTopLogoFlex: {
-    paddingRight: "1rem",
-    ".mobile &": {
-      minWidth: "200px",
-      paddingRight: 0,
-      justifyContent: "center",
+  footerTopLeftContainer: {
+    flex: "0 0 50%",
+    maxWidth: "100%",
+    minWidth: "100%",
```
Comment (braxex) [inline services/ui-src/src/components/layout/Footer.tsx:122]: Question: Could we just use `width: "100%"` here? Same question with other container below.

File: services/ui-src/src/components/layout/Footer.tsx [hunk 3]
```diff
@@ -105,57 +114,93 @@ const sx = {
...
-    ".mobile &": {
-      maxWidth: "10.125rem",
+  footerCMSBrandingLeft: {
+    display: "flex",
+    justifyContent: "center",
+    alignItems: "center",
+    maxWidth: "100%",
+    padding: "0 1rem",
+    minWidth: "100%",
+    width: "100%",
```
Comment (braxex) [inline services/ui-src/src/components/layout/Footer.tsx:154]: Question: Seems unnecessary to set all these widths the same. Can probably just set `width` to 100 and then auto.

File: services/ui-src/src/components/layout/Footer.tsx [hunk 4]
```diff
@@ -105,57 +114,93 @@ const sx = {
...
-    marginY: "1.75rem",
-    justifyContent: "center",
+    maxWidth: "12.5rem",
+    marginLeft: "1.25rem",
+  },
+  hhsCopyText: {
+    display: "inline-block",
+    textAlign: "left",
+    verticalAlign: "top",
+    fontSize: 14,
```
Comment (braxex) [inline services/ui-src/src/components/layout/Footer.tsx:188]: Nit: Use theme value `"sm"` here.

**Review by braxex — APPROVED** [review]
Overall looks great. A couple nits and questions, but nothing blocking.

File: services/ui-src/src/components/layout/Footer.tsx (hunk 3 above)
Comment (ntsummers1) [reply services/ui-src/src/components/layout/Footer.tsx:154]: Can do away with width I believe. Need minWidth and maxWidth though for how diff browsers can handle it. minWidth in particular though is the true one we could keep

**Review by braxex — APPROVED** [review]

**Pull request merged**